modules/manage/pages/manage-xdcr/replicate-using-scopes-and-collections.adoc

```diff
@@ -0,0 +1,178 @@
+= Replicate Using Scopes and Collections
+
+[abstract]
+XDCR can be performed with reference to scopes and collections within source and target buckets.
+
+[#understanding-scopes-collections-and-replication]
+== Understanding Scopes, Collections, and Replication
+
+As described in xref:learn:data/scopes-and-collections.adoc[Scopes and Collections], scopes and collections are containers for data within a bucket.
+A bucket can contain up to 100 scopes; and a scope can contain up to 1000 buckets.
+
+As well as specifying a source bucket and a target bucket, XDCR can specify scopes and collections within those buckets.
+This allows data to be reassigned from one collection to another.
+It also provides a convenient way of _migrating_ data that currently resides in legacy buckets that do not support (or have not yet been established with) administrator-defined scopes and collections, into buckets, on a Version 7.0+ Couchbase-Server cluster, that _do_ contain administrator-defined scopes and collections.
+For more information, see xref:learn:clusters-and-availability/xdcr-overview.adoc#xdcr-with-scopes-and-collections[XDCR with Scopes and Collections].
+
+[#examples-on-this-page]
+== Examples on this Page
+The examples provided on this page &#8212; xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#migrate-data-to-a-collection[Migrate Data to a Collection] and xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#replicate-data-between-collections[Replicate Data Between Collections] &#8212; demonstrate how XDCR can be performed with reference to scopes and collections within source and target buckets, using Couchbase Web Console.
+The examples assume the following:
+
+* A single-node cluster named `10.144.210.101` has been created, and features the `travel-sample` bucket.
```

Comment:
Right. So, unfortunately, this is the standard naming currently used throughout all the management doc, not just for XDCR, but for every feature - including UI, CLI, and REST examples. I can do this, but it might be better to wait until we are through the current Beta, first. The new standard I use for naming should probably be adopted elsewhere in the docs also.